### DISCUSSION TITLE
Major version updates and PRs

### DISCUSSION THREAD

**Discussion by biohazd** [renovatebot/renovate#30148] [Suggest an Idea]
### Tell us more.

Do you happen to know if it's possible to configure Renovate to not skip major versions when raising PRs?

E.g. if my current version is v25.0.0 and there is both v26.0.0 and v27.0. 0 available I only want Renovate to create a PR for v26.0.0

I can get close with:
"separateMajorMinor": true,
"separateMultipleMajor": true

But this ends up creating two PRs, one from v25=>v26 and another from v25=>v27 (I don't want this one until the other is merged)

**Comment by rarkins** [comment]
Not possible
